lib/src/base-styles/_nx-btn.scss

```diff
@@ -101,6 +101,34 @@
       height: 16px;
     }
   }
+
+  &--small {
+    font-size: var(--nx-font-size-s);
+    padding: var(--nx-spacing-base) var(--nx-spacing-3x);
+    margin: 0 var(--nx-spacing-3x) 0 0;
+    border-color: var(--nx-color-small-button-border);
+    color: var(--nx-color-small-button-text);
```

Comment:
Related to https://github.com/sonatype/sonatype-react-shared-components/pull/1220#issuecomment-2730338949, these colors should depend on whether it's a primary, secondary, etc button.

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/1a8d7d3e82a75132d6e193e3bae6877b22181660